### PR TITLE
Kyle/app 20 create icon handler

### PR DESCRIPTION
**Pull request by Kyle-Mie**
See /testicon route for a demo of the functionality. This allows the programmer to dynamically resize icons and change the color without the need to import the SVG paths for each variation.

### CODE REVIEW

File: APolitcalApp/src/views/IconTestView.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+import IconBase from "../components/icons/IconBase.vue";
+import IconApprove from "../components/icons/IconApprove.vue";
+
+export default{
```
Comment (Colby-M) [inline APolitcalApp/src/views/IconTestView.vue]: This looks like options api, make sure you are using the composition api instead. Basically Vue 2 vs Vue 3

**Review by Colby-M — COMMENTED** [review]
Make sure it's composition then should be good

**Comment by Kyle-Mie** [comment]
@Colby-M Changed it. I think i changed it to composition API but I'm not exactly sure if I did it right. Let me know if I goofed it up.

**Review by Colby-M — APPROVED** [review]

**Comment by Colby-M** [comment]
Looks good

**Pull request merged**